src/Text/Regex/Base/Context.hs

```diff
@@ -11,29 +11,29 @@ Stability   :  experimental
 Portability :  non-portable (MPTC+FD)
 
 This is a module of instances of 'RegexContext' (defined in
-Text.Regex.Base.RegexLike).  Nothing else is exported.  This is
-usually imported via the Text.Regex.Base convenience package which
-itself is re-exported from newer Text.Regex.XXX modules provided by
-the different regex-xxx backends.
+"Text.Regex.Base.RegexLike").  Nothing else is exported.  This is
+usually imported via the "Text.Regex.Base" convenience package which
+itself is re-exported from newer @Text.Regex.XXX@ modules provided by
+the different @regex-xxx@ backends.
 
 These instances work for all the supported types and backends
-interchangably.  These instances provide the different results that
-can be gotten from a match or matchM operation (often via the @=~@ and
-@=~~@ operators with combine @makeRegex@ with @match@ and @matchM@
-respectively).  This module name is Context because they operators are
-context dependent: use them in a context that expects an Int and you
-get a count of matches, use them in a Bool context and get True if
+interchangeably.  These instances provide the different results that
+can be gotten from a 'match' or 'matchM' operation (often via the @=~@ and
+@=~~@ operators with combine 'makeRegex' with 'match' and 'matchM'
+respectively).  This module name is @Context@ because they operators are
+context dependent: use them in a context that expects an 'Int' and you
+get a count of matches, use them in a 'Bool' context and get 'True' if
 there is a match, etc.
 
-@RegexContext a b c@ takes a regular expression suppied in a type 'a'
-generated by 'RegexMaker' and a target text supplied in type 'b' to a
-result type 'c' using the 'match' class function.  The 'matchM' class
+@'RegexContext' a b c@ takes a regular expression suppied in a type @a@
+generated by 'RegexMaker' and a target text supplied in type @b@ to a
+result type @c@ using the 'match' class function.  The 'matchM' class
 function works like 'match' unless there is no match found, in which
 case it calls 'fail' in the (arbitrary) monad context.
 
-There are a few type synonyms from RegexLike that are used here:
+There are a few type synonyms from "RegexLike" that are used here:
```

Comment:
Double quotes?

Reply:
Expanded the module name to full qualification to make the link work.